src/router.js

```diff
@@ -333,6 +335,16 @@ export class Router {
     return undefined;
   }
 
+  _transformTitle(title: string) {
+    if (this.transformTitle) {
+      return this.transformTitle(title);
+    }
+    if (this.parent) {
+      return parent.transformTitle(title);
+    }
+    return title;
+  }
+
```

Comment:
We could probably remove _transformTitle, and instead let transformTitle have the default implementation before/if it is overridden. That way we won't have any confusion if internal objects should be using _transformTitle or transformTitle.